contracts/TicketWallet.sol

```diff
@@ -54,6 +54,18 @@ contract TicketWallet is ERC721Token, Pausable {
    */
   Ticket[] public tickets;
 
+  /**
+   * Retailers contract address
+   */
+  address public retailers;
```

Comment:
Can we call this retailersIndex or something like that? I was going to use retailers as a mapping to keep track of who sold which tickets.

Reply:
`retailerStore`?